### PR TITLE
B 19751

### PR DESCRIPTION
**Pull request by CoryKleinjanCACI**
## Summary

This task give the TOO role that ability to add shipments of any kind. The TOO will not have the ability to configure advance requests for PPMs.

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Agility acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the Office role
2. Login as a TOO
3. Select a move
4. navigate to the move details tab

### Repeat steps 5-6 for each shipment type (HHG, PPM, NTS, NTS-release)
5. Add a shipment
6. Verify the  shipment exists

### Frontend

- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://transcom.github.io/mymove-docs/docs/adrs/Browser-Support/#minimum-browser-requirements) (Chrome, Firefox, Edge).
- [ ] There are no new console errors in the browser devtools.
- [ ] There are no new console errors in the test output.
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.
- [ ] This change meets the standards for [Section 508 compliance](https://www.ssa.gov/accessibility/andi/help/install.html).

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/getting-started/development/logging).
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/backend/guides/golang-guide#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#zero-downtime-migrations).
- [ ] Have been communicated to #g-database.
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#secure-migrations).

## Screenshots
Screenshot 2024-06-17 at 12 03 57 PM

### CODE REVIEW

File: src/pages/Office/MoveTaskOrder/MoveTaskOrder.jsx [hunk 1]
```diff
@@ -123,7 +123,6 @@ export const MoveTaskOrder = (props) => {
   const [estimatedWeightTotal, setEstimatedWeightTotal] = useState(null);
   const [estimatedPPMWeightTotal, setEstimatedPPMWeightTotal] = useState(null);
   const [, setSubmittedChangeTime] = useState(Date.now());
-
   const nonShipmentSections = useMemo(() => {
```
Comment (antgmann) [inline src/pages/Office/MoveTaskOrder/MoveTaskOrder.jsx:126]: I don't see this change on the INT PR... REJECTED.

**Review by antgmann — APPROVED** [review]
Just kidding, everything looks good, commit hashes and changes match.

File: src/pages/Office/MoveTaskOrder/MoveTaskOrder.jsx (hunk 1 above)
Comment (r-mettler) [reply src/pages/Office/MoveTaskOrder/MoveTaskOrder.jsx:126]: Yeah I don't know how I feel about you trying to sneak in a line removal 🙅‍♂️

**Review by r-mettler — APPROVED** [review]
Changes same as INT, minus the line removal, but I guess I will let that slide 😄

File: src/pages/Office/MoveTaskOrder/MoveTaskOrder.jsx (hunk 1 above)
Comment (CoryKleinjanCACI) [reply src/pages/Office/MoveTaskOrder/MoveTaskOrder.jsx:126]: Im like 99% sure that was the linter, but if it breaks something at least I can say you approved it.

**Pull request merged**